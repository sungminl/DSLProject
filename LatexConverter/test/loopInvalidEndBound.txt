&a = 1

LOOP &a FROM 1 - 1 TO log(0)
{
MATRIX
ROW: 2, COL: 4(&a)
{
2 * &a|
&a / 2|
}
}
