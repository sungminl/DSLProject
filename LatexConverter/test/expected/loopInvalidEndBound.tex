The program could not successfully complete.
A runtime error was encountered:
Invalid end bound equation for loop.
